environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - conda-forge
 dependencies:
   - conda-forge::python=3.9
-  - pandas
+  - pandas<2.0
```

Comment:
I would recommend having the `conda-forge::` notation for all dependencies in the environment to stay consistent as well as this is the standard it seems for all projects now.